app/helpers/admin_helper.rb

```diff
@@ -16,4 +16,28 @@ def all_emails_associated_with_a_user(induction_record)
   def on_admin_npq_application_page?
     request.path.starts_with?("/admin/npq/applications")
   end
+
+  def html_list(values)
+    return nil if values.empty?
+
+    tag.ul(class: %w[govuk-list]) { safe_join(values.map { |v| tag.li(v) }) }
+  end
+
+  def induction_programme_friendly_name(name, short: false)
```

Comment:
Is a helper the right place for this? Didn't feel like it was doing enough _logic_ for it to be in a service class of its own.